app/uk/gov/hmrc/pensionschemereturn/audit/ApiAuditUtil.scala

```diff
@@ -120,71 +120,11 @@ class ApiAuditUtil @Inject() (auditService: AuditService) extends Logging {
   ): PartialFunction[Try[Option[PsrSubmissionEtmpResponse]], Unit] = {
     case Success(optPsrSubmissionEtmpResponse) =>
       logger.info(s"PsrGetAuditEvent ->> Status: ${Status.OK}")
-      auditService.sendEvent(
```

Comment:
Unless logs are meaningful, `firePsrGetAuditEvent` can be potentially removed too.